cmd/juju/application/removeunit.go

```diff
@@ -242,10 +268,22 @@ func (c *removeUnitCommand) removeUnitsDeprecated(ctx *cmd.Context, client Remov
 }
 
 func (c *removeUnitCommand) removeUnits(ctx *cmd.Context, client RemoveApplicationAPI) error {
+	var maxWait *time.Duration
+	if c.Force {
+		if c.NoWait {
+			zeroSec := 0 * time.Second
+			maxWait = &zeroSec
+		} else {
+			oneMin := 1 * time.Minute
```

Comment:
Would prefer that the wait time in params is a pointer. If not set, the server gets to decide what to use. Save all the different CLI clients from needing to specify it. And there's no common const used anyway.

Reply:
It is a pointer here but I'll change to pass nil when --no-wait is false.